components/identity.oauth.uma.resource.endpoint/src/main/resources/resourceregistration.yaml

```diff
@@ -0,0 +1,302 @@
+swagger: '2.0'
```

Comment:
If we put this, we may need to add license header for this as well.

Reply:
Done.

Reply:
Done.